features/step_definitions/smokey_steps.rb

```diff
@@ -31,7 +31,7 @@
 
   page.driver.browser.agent.add_auth(base_host, ENV['AUTH_USERNAME'], ENV['AUTH_PASSWORD'])
 
-  visit url
+  @response = get_request(url, default_request_options)
```

Comment:
Thanks, I find it really difficult to figure out how (our usage of?) Cucumber/Capybara/rest-client etc all tie together. I've removed this commit and started using a `Capybara::Session` instead.